user-profiles.md

```diff
@@ -0,0 +1,36 @@
+# User Profiles
+
+The purpose of this document is to aide in the development of Helm features by evaluating Helm features against those whole will be using them.
```

Comment:
s/aide/aid --  I suspect you meant to use the verb form instead of the noun form.